code/game/objects/items/devices/laserpointer.dm

```diff
@@ -0,0 +1,126 @@
+/obj/item/weapon/gun/energy/laser_pointer
+	name = "laser pointer"
+	desc = "Don't shine it in your eyes!"
+	icon = 'icons/obj/device.dmi'
+	icon_state = "pointer"
```

Comment:
No sprites as well.

Reply:
the sprites were never removed from code

https://i.imgur.com/6JHvVT9.jpg